### ISSUE TITLE
Navigation Menu design Time

### ISSUE THREAD

**Issue by zacbryant2010** [steelvelveteen/Tafe17S105#2]
Hey Everyone, i would like to know everyone's opinions on creating our navigational menu, and if to implement it as an individual branch, or to update it in the master File
I myself am leaning towards the idea of creating a navigational form branch at the completion of the previous branches- Zac Bryant

**Comment by steelvelveteen** [comment]
I agree. First as a branch, we keep it there. When we all finish with out corresponding features we see if it works and if it does, merge it all together.

**Comment by tonygags** [comment]
Yes Agreed. Keep it as a separate branch so it can be modified independently and merged when needed.

**Comment by Wang097** [comment]
Thanks guys for your opinions, I agree with merging after completing the branches.
